### PR TITLE
chore: migrate to latest version of nostr-tools

### PR DESCRIPTION
**Pull request by im-adithya**
Title

### CODE REVIEW

**Review by rolznz — APPROVED** [review]
utACK

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "light-bolt11-decoder": "^3.2.0",
     "lint-staged": "^14.0.0",
     "microbundle": "^0.15.1",
-    "nostr-tools": "^1.17.0",
+    "nostr-tools": "^2.10.4",
```
Comment (rolznz) [inline package.json:62]: is this supposed to be a dev dependency? but it seems to work

Comment (im-adithya) [reply package.json:62]: Yeah same reaction, we only use it in examples, so just a simple change in here

Comment (rolznz) [reply package.json:62]: Ah, cool. Thanks!

**Pull request merged**